runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerConfig.java

```diff
@@ -94,11 +94,11 @@ default int statTimeout()
 		position = 5,
 		keyName = "highlightTargets",
 		name = "Highlight Targets",
-		description = "Highlight monsters you can kill for your current slayer assignment"
+		description = "Highlight the hull or tiles of monsters you can kill for your current slayer assignment"
 	)
-	default boolean highlightTargets()
+	default SlayerTaskHighlightMode highlightTargets()
 	{
-		return false;
+		return SlayerTaskHighlightMode.NONE;
```

Comment:
Can anything be done here to allow me to pull in their previous default, since this would be changing the type stored on the config? If it was previously `true` we would want it to be `SlayerTaskHighlightMode.HULL`

Reply:
If by previous default you mean previous setting, then, you could write code to migrate from the old value to the new value (assuming they have two different keys), which we will do occasionally. However this feature is little-used and so I probably wouldn't bother with it. You have set the default value to be the same as the existing default value (off) so this PR will not cause a behavior change.

If I recall correctly the config manager has code to deal with this (currently what you have is you are trying to unserialize a true/false into a boolean value, which is not valid) and I think it will force-reset the config value to its default.

Reply:
So would I be fine to just leave it as is and have it force-reset?

Reply:
I'd have to check but I think so. If you run the client you should see it hit this `log.debug("Setting default configuration value for {}.{} to {}", group.value(), item.keyName(), defaultValue);` and override the default value on the config after being unable to deserialize it.

Reply:
Yup, I did in fact see that, should be good